frontend/src/components/HomePage.js

```diff
@@ -198,30 +198,30 @@ class HomePage extends React.Component<{
                             </Grid>
                             <Grid item xs={12} sm={8} className={classes.textBox}>
                                 <Typography>
-                                    For more <b>advanced users</b>, the CodaLab Wiki has a wealth of
+                                    For more <b>advanced users</b>, the CodaLab documentation has a wealth of
                                     information about many aspects of using the platform:
                                 </Typography>
                                 <Typography>
                                     <Table padding='dense'>
                                         {this.renderTableItem(
                                             'Workflow',
                                             'Use CodaLab in your daily research.',
-                                            'https://github.com/codalab/codalab-worksheets/wiki/Workflow',
+                                            'https://codalab.readthedocs.io/en/latest/Workflow',
```

Comment:
Ideally, this would point to the version of the readthedocs corresponding to the this branch...is that worth setting up?

Reply:
Readthedocs usually lets you just configure docs published for a set number of branches. We probably won't be publishing docs for each PR / branch.